### PR TITLE
Derive providertype from org

### PR DESCRIPTION
**Pull request by kosiakkatrina**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -21,8 +20,8 @@
     t.string "reset_password_token"
     t.datetime "reset_password_sent_at"
     t.datetime "remember_created_at"
-    t.datetime "created_at", precision: 6, null: false
```
Comment (baarkerlounger) [inline db/schema.rb]: These make me nervous. Can you find out where they come from or revert?

File: app/models/constants/case_log.rb [hunk 2]
```diff
@@ -1125,4 +1125,9 @@ module Constants::CaseLog
                             "Other"].freeze
 
   OPTIONAL_FIELDS = %w[postcode_known la_known first_time_property_let_as_social_housing tenant_code propcode].freeze
+
+  PROVIDER_TYPE = {
```
Comment (smeny) [inline app/models/constants/case_log.rb]: We could reuse the organisation constants here

File: db/schema.rb [hunk 3]
```diff
@@ -191,6 +190,7 @@
     t.decimal "tshortfall", precision: 10, scale: 2
     t.decimal "chcharge", precision: 10, scale: 2
     t.integer "declaration"
+    t.integer "providertype"
```
Comment (smeny) [inline db/schema.rb:193]: Why is this not fetched from the organisation directly? I think that adding it like this creates a duplication where we could have the organisation and the case logs having different values.

File: db/schema.rb (hunk 1 above)
Comment (kosiakkatrina) [reply db/schema.rb]: It had precision when it was created I'll revert it

**Pull request closed without merging**